app/uk/gov/hmrc/plasticpackagingtax/registration/services/GRSRedirections.scala

```diff
@@ -50,16 +50,18 @@ trait GRSRedirections extends I18nSupport {
   ): Future[String] =
     ukCompanyGrsConnector.createJourney(incorpEntityGrsCreateRequest(callbackUrl), grsUrl)
 
-  def getPartnershipRedirectUrl(grsUrl: String, callbackUrl: String)(implicit
-    request: JourneyRequest[AnyContent],
-    hc: HeaderCarrier
-  ): Future[String] =
+  def getPartnershipRedirectUrl(
+    grsUrl: String,
+    callbackUrl: String,
+    businessVerification: Boolean = true
+  )(implicit request: JourneyRequest[AnyContent], hc: HeaderCarrier): Future[String] =
     partnershipGrsConnector.createJourney(
       PartnershipGrsCreateRequest(callbackUrl,
                                   Some(request2Messages(request)("service.name")),
                                   appConfig.serviceIdentifier,
                                   appConfig.signOutLink,
-                                  appConfig.grsAccessibilityStatementPath
+                                  appConfig.grsAccessibilityStatementPath,
+                                  businessVerificationCheck = businessVerification
```

Comment:
Should this be `businessVerificationCheck = false` too or am I talking rubbish here?

Reply:
No, because we need businessVerificationCheck to be true for partnership journey and false for partners journey.